pandas/tests/test_categorical.py

```diff
@@ -1554,18 +1554,6 @@ def test_deprecated_labels(self):
             res = cat.labels
         self.assert_numpy_array_equal(res, exp)
 
-    def test_deprecated_levels(self):
```

Comment:
Afai understand the comments, this was exactly what happend already: the comments were how to create the test pickle file, the tests loaded it and compared it with the equivalent categorical as it would be created today:

https://github.com/gfyoung/pandas/blob/14e2b7412b85606898d39e30dd8c7d00a866d5c8/pandas/tests/test_categorical.py#L4395

```
 def test_pickle_v0_14_1(self):

        # we have the name warning
        # 10482
        with tm.assert_produces_warning(UserWarning):
            cat = pd.Categorical(values=['a', 'b', 'c'],
                                 categories=['a', 'b', 'c', 'd'],
                                 name='foobar', ordered=False)
        pickle_path = os.path.join(tm.get_data_path(),
                                   'categorical_0_14_1.pickle')
        # This code was executed once on v0.14.1 to generate the pickle:
[...]
    def test_pickle_v0_15_2(self):`
[...]
``